### ISSUE TITLE
ESPbootloader crash/restarts on new "dev" branch

### ISSUE THREAD

**Issue by delinend** [sebastianhodapp/ESPbootloader#8]
I'm testing ESPbootloader on the new NodeMCU "dev" brach, and it crash/restarts just after the "Please connect to: 192.168.4.1" message, and I can't figure out, what the problem is. There is no problems with using ESPbootloader, on the "master" branch. The reason using the "dev" branch is, that want to test the new RTCtime and SNTP.

Btw. I user Marcel's NodeMCU custom builder: http://frightanic.com/nodemcu-custom-build/index.php

Get available APs

> Preparing HTML Form
> Setting up Wifi AP
> Setting up webserver
> Setting up Webserver done. Please connect to: 192.168.4.1
> ò˜Öò 
> ¤„ÿ1ä)]ô1ä)Í!ÜNÿÐ

NodeMCU custom build by frightanic.com
                             branch: dev
                             commit: 5d036c461bda0f50b4ba3e16ceb1fd4b80f202ee
                             SSL: false
                             modules: node,file,gpio,wifi,net,i2c,tmr,uart,u8g,rtctime,sntp
build                  built on: 2015-11-09 13:32
powered by Lua 5.1.4 on SDK 1.4.0

Best regards

**Comment by rwkiii** [comment]
I also use a build generated by nodemcu-build.com and am having the constant rebooting/resetting issue, I also see the garbage chars but presumed those were normal output at a different baud rate being sent by ESP.

You mentioned this 'crash' doesn't happen with the master build here, but that's what I'm using.

Did you ever resolve this problem?

**Comment by delinend** [comment]
Yes, I solved it with updating the Flash memory chip, on my ESP-07, to a 4Mbyte chip. 
A 512Kbyte chip was to small for me.

**Comment by marcelstoer** [comment]
> I also use a build generated by nodemcu-build.com and am having the constant rebooting/resetting issue

You should check out http://nodemcu-build.com/faq.php#boot-failure
